src/SBP_coefficients/WilliamsDuru2024.jl

```diff
@@ -0,0 +1,766 @@
+                                                      T(-1726424775466658030439702201256805556456624//3991094794532025114962898069922411226475337),
+                                                      T(-4878663543549720465771003262056834494382336//99777369863300627874072451748060280661883425),
+                                                      T(2465969237005876777755627421935472827876576//99777369863300627874072451748060280661883425), )),
+            DerivativeCoefficientRow{T,1, 8}(SVector( T(-35741686713503424823053247//1274365888916385703504957136),
+                                                      T(24341883735526123464906853//955774416687289277628717852),
+                                                      T(-5180106064713036907224775//79647868057274106469059821),
+                                                      T(71057867556242896847948430//79647868057274106469059821),
+                                                      T(-2501516650966057622106220669//3823097666749157110514871408),
+                                                      T(-10392298878790214848122661//318591472229096425876239284),
+                                                      T(-109408685199711207626220261//637182944458192851752478568),
+                                                      T(2716694214891248947829397//79647868057274106469059821), )),
+            DerivativeCoefficientRow{T,1, 9}(SVector( T(389387454207582946160914171//15024124021976694625393539385),
+                                                      T(-52575461023873067818412747//3004824804395338925078707877),
+                                                      T(-355584433607894620879408295//3004824804395338925078707877),
+                                                      T(-189974696081312018416760220//3004824804395338925078707877),
+                                                      T(3748169214543475593647140365//3004824804395338925078707877),
+                                                      T(-29791922049731222907256317667//30048248043953389250787078770),
+                                                      T(117651741517270046035097757//3004824804395338925078707877),
+                                                      T(-914381078809660182342952743//6009649608790677850157415754),
+                                                      T(90818888188769860097268444//3004824804395338925078707877), )),
+        )
+        upper_coef_plus = SVector(T(205//143), 
+                                  T(-873//2860), 
+                                  T(-133//4290), 
+                                  T(3//130), )
+        central_coef_plus = T(-889//858)
+        lower_coef_plus = SVector(T(57//1430), 
+                                  T(-443//2860), 
+                                  T(2//65), )
```

Comment:
I haven't read the paper in detail yet. Looking at the arXiv version, these coefficients should be the ones from the table in the appendix, Section A.1, correct? If so, could you please explain the differences?

Reply:
The 4th order accurate interior stencil was updated since the 2021 version was uploaded to archive. The coefficients I used should match the version on SIAM.

Reply:
Thanks!